Source/WebKit/NetworkProcess/storage/BackgroundFetchStoreImpl.cpp

```diff
@@ -382,7 +382,7 @@ void BackgroundFetchStoreImpl::getBackgroundFetchState(const String& backgroundF
             return;
         }
         auto information = fetch->information();
-        callback(BackgroundFetchState { key.topOrigin(), key.scope(), fetch->identifier(), fetch->options(), information.uploadTotal, information.uploaded, information.downloadTotal, information.downloaded, information.result, information.failureReason, fetch->isActive() });
+        callback({ { key.topOrigin(), key.scope(), fetch->identifier(), fetch->options(), information.downloadTotal, information.downloaded, information.uploadTotal, information.uploaded, information.result, information.failureReason, fetch->isActive() } });
```

Comment:
The last argument should be `fetch->isPaused()`

Reply:
Right, I'll beef up the test to cover this